### ISSUE TITLE
Overhead lights don't behave as if overhead

### ISSUE THREAD

**Issue by Regularitee** [CleverRaven/Cataclysm-DDA#31064]
Mounting overhead lights from emergency vehicles (the red and blue flashing ones associated with sirens) are not considered roof-mounted for lighting purposes. Instead, they behave as if they were inside the driver's compartment itself: like normal light sources, they will be blocked by curtains or vehicle boards. Which one would not expect from a roof-mounted light source.

**Comment by Night-Pryanik** [comment]
Closing in favor of #34764.
